osquery/tables/system/darwin/crash_logs.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <boost/algorithm/string/predicate.hpp>
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/trim.hpp>
+#include <boost/format.hpp>
+#include <boost/regex.hpp>
+
+#include "osquery/tables/system/system_utils.h"
+#include <osquery/core/conversions.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+namespace fs = boost::filesystem;
+namespace alg = boost::algorithm;
+namespace osquery {
+namespace tables {
+const std::string kDiagnosticReportsPath = "/Library/Logs/DiagnosticReports";
+const std::map<std::string, std::string> kCrashDumpKeys = {
```

Comment:
For both the above `kDiagnosticReportsPath` and this `kCrashDumpKeys`, let's add some doxygen comments.
Feel free to use `///` on the proceeding line for a brief statement about the variable.